.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,23 @@
+<!-- 
+Please explain WHAT you changed.
+
+The title should be descriptive, for example:
+
+* *Fixed a typo in the apikeypermissions.md page*
+* *Added the maximum number of domain whitelabels you can create to domains.md*
+* *Fixing the number of days a batch id is valid in scheduling_parameters.md*
+
+If this PR fixes an issue, please reference the issue number as well.
+
+Fill out this form in the body:
+-->
+
+**Description of the change**:
+
+If you have questions, please send an email [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.
+
+<!-- 
```

Comment:
We added this in our template, and it's been really useful because it triggers githubs functionality to automatically close the issue when the pull request is merged:
<!-- 
If this pull request closes an issue, add in the issue number here 
-->
Closes #(issue number)